flink-connector-mongodb-cdc/src/test/java/com/ververica/cdc/connectors/mongodb/MongoDBSourceExampleTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ververica.cdc.connectors.mongodb;
+
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import org.apache.flink.streaming.api.functions.source.SourceFunction;
+
+import com.ververica.cdc.debezium.JsonDebeziumDeserializationSchema;
+
+/** Example Tests for {@link MongoDBSource}. */
+public class MongoDBSourceExampleTest {
```

Comment:
Thanks @vajaw for the contribution, could you refer `MySqlSourceExampleTest` to make `MongoDBSourceExampleTest` be a unit test without any preparation.

Reply:
Ok, i can

Reply:
Hi @vajaw, are you going to push a commit to resolve this comment?

1.  Extends `MongoDBTestBase` to setup a test MongoDB instance.
2. Prepare some records by: `String database = executeCommandFileInSeparateDatabase("inventory");`
3. Connect the MongoDB instance then print the data.